src/uct/ib/base/ib_md.c

```diff
@@ -1764,6 +1764,12 @@ static void uct_ib_md_vfs_init(uct_md_h md)
     }
 }
 
+static ucs_status_t uct_ib_md_get_flags(uct_ib_md_t *md, uint64_t *flags)
+{
+    *flags = 0;
+    return UCS_OK;
+}
+
```

Comment:
maybe just add `flags` field to `uct_ib_md_t` and set the corresponding flags from md opening routines?
Imo, no need to introduce new function in ops

Reply:
as I can see there is no way to update md flags based on md_ops selected

Reply:
discussed offline, should be possible to use flag instead of new func